### PR TITLE
WordPress 5.7 jQuery Compatibility 3.5.X

### PR DESCRIPTION
**Pull request by bordoni**
🎫 [TCMN-99]
---
No particular asset here, testing no necessarily required here, no modules were changed in behavior. And will be covered by TEC/ET updated.

[TCMN-99]: https://theeventscalendar.atlassian.net/browse/TCMN-99

### CODE REVIEW

**Review by sc0ttkclark — DISMISSED** [review]
Solid improvements with the docs

File: src/resources/js/dropdowns.js [hunk 1]
```diff
@@ -79,7 +79,7 @@ var tribe_dropdowns = window.tribe_dropdowns || {};
 	 */
 	obj.matcher = function ( params, data ) {
 		// If there are no search terms, return all of the data
-		if ( $.trim( params.term ) === '') {
+		if ( params.term.trim() === '') {
```
Comment (sc0ttkclark) [inline src/resources/js/dropdowns.js:82]: `$.trim()` usage might be in selectWoo too

File: src/resources/js/validation.js [hunk 2]
```diff
@@ -14,7 +16,7 @@ tribe.validation = {};
 	/**
 	 * Object containing all the selectors for Validation
 	 *
-	 * @since  4.7
+	 * @since  4.71
```
Comment (paulmskim) [inline src/resources/js/validation.js:19]: 4.7.1? or 4.71?

Comment (sc0ttkclark) [reply src/resources/js/validation.js:19]: @bordoni I think this was meant to be 4.7.1?

```suggestion
	 * @since  4.7.1
```

**Review by sc0ttkclark — COMMENTED** [review]
Two Qs left to resolve here

**Pull request merged**